account_reversal/README.rst

```diff
@@ -12,9 +12,30 @@ Also add on account entries:
 * a checkbox and filter "to be reversed"
 * a link between an entry and its reversal entry
 
+Odoo v9c include an action similar (overwritten by this addon), but with less
+features, for instance:
+
+* Allowing inheritance
+* Options like prefix (for entry and entry line), post and reconcile.
+* Create a link between the entry and its reversal
+* Mark entries to be reversed in the future.
+
 Usage
 =====
 
+As in Odoo v9c, if you select an entry from Accounting > Adviser > Journal Entries,
+then an action menu 'Reverse Entries' is available. If clicked, then a wizard
+allows user to select Reversal Date, Reversal Journal, Prefix, Post and Reconcile.
+
+* If no Reversal Journal is selected, then the same journal is used
+* If Post is True, then reversal entry will be posted else it will be leaved
+as a draft entry
```

Comment:
* If Post is True, then the reversal entry will be posted else it will be left
  as a draft entry